Backend/src/API/REST/routes/reset_institution_pictures.ts

```diff
@@ -0,0 +1,72 @@
+import check_auth from "../middleware/check_auth";
+import { ExtRequest } from "../../../definitions/ext_request";
+import express, { Response } from "express";
+import processImageUpload from "../../../util/processImageUpload";
+import InstitutionService from "../../../database/services/institution.service";
+import { UpdateInstitutionDTO } from "../../../database/dtos/institution.dto";
+import { IInstitution } from "../../../database/models/institution";
+import s3 from "../../../util/s3";
+import { S3 } from "aws-sdk";
+
+async function retrieveInstitutionFromRequest(req: ExtRequest, res: Response): Promise<string | null> {
+    if (!req.user_id) {
```

Comment:
This is already checked by `check_auth`  middleware, so I don't think we should add this check also here.